portfolio/src/main/webapp/style.css

```diff
@@ -56,13 +79,46 @@ img:hover {
     box-shadow: 0 0 2px 1px white;
 }
 
+li {
+    list-style-type:none;
+    margin-top: 15px;
+}
+
+#name-box {
+    background: url(images/better.jpg) center center no-repeat fixed;
+    border: 1px solid white;
+    outline: none;
+    resize: none;
+    color: floralwhite;
+    width: 12em;
+    height: 1.75em;
+}
+
 #random-quote-button {
     border-radius:10px;
     color: floralwhite;
-    background-color: black;
-  
+    background-color: black; 
 }
 
 #quote-container {
     margin-top: 15px;
+}
+
+#text-box {
+    background: url(images/better.jpg) center center no-repeat fixed;
+    border: 1px solid white;
+    outline: none;
+    resize: none;
+    color: floralwhite;
```

Comment:
You can separate out this part into a rule a common rule for these common things:
#text-box, #name-box {
   *common stuff*
}
#name-box {
   *name only*
}
#text-box {
   *text box only*
}